test/modules/TestResultCoordinator/report/IResults.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator
+{
+    using System;
+    using System.Threading.Tasks;
+
+    // TODO: Replace IResults with IAsyncEnumerable when project is moved to .Net Core 3.1
+    public interface IResults<T> : IDisposable
```

Comment:
rename to ITestResultSource

Reply:
implements IEnumerator

Reply:
I renamed to ITestResults, because the interface doesn't know about the source, it just returns results